### ISSUE TITLE
libfaketime

### ISSUE THREAD

**Issue by haqistan** [torbsd/openbsd-ports#1]
Write an OpenBSD port for [libfaketime](https://github.com/wolfcw/libfaketime)
which is used by the client-side install script in TBB.

**Comment by haqistan** [comment]
Not so sure we need this after all, I think it is just used to zero out (effectively)
timestamps in the resulting bundle so that they are reproducible.  OpenBSD packages do
this in another way.

**Comment by thesamesam** [comment]
You are correct. Gitian is the software used for reproducible builds. I'm not familiar with OpenBSD's port reproducibility, but it may be sensible to verify it works before writing this off the list.

ref.
https://www.torproject.org/docs/faq.html.en#Timestamps
https://blog.torproject.org/blog/deterministic-builds-part-two-technical-details

> Millisecond and below timestamps are not fixed by libfaketime
> **Gitian relies on libfaketime** to set the clock to a fixed value to deal with embedded timestamps in archives and in the build process. However, in some places, Firefox inserts millisecond timestamps into its supporting libraries as part of an informational structure. We simply zero these fields.
